api/lib/infrastructure/repositories/certification-candidate-repository.js

```diff
@@ -58,6 +59,20 @@ module.exports = {
       .then((results) => bookshelfToDomainConverter.buildDomainObjects(CertificationCandidateBookshelf, results));
   },
 
+  getBySessionIdAndUserId({ sessionId, userId }) {
+    return CertificationCandidateBookshelf
+      .where({ sessionId, userId })
+      .fetch({ require: true })
+      .then((result) => bookshelfToDomainConverter.buildDomainObject(CertificationCandidateBookshelf, result))
+      .catch((error) => {
+        if (error instanceof CertificationCandidateBookshelf.NotFoundError) {
+          throw new NotFoundError(`Not found certification-candidate for sessionId ${sessionId} and userId ${userId}`);
```

Comment:
`Not found certification-candidate for` ce n'est pas très correct comme tournure de phrase.

Reply:
Merci, c'est corrigé